samples/angular-app/package.json

```diff
@@ -19,7 +19,9 @@
     "@angular/platform-browser": "~9.1.6",
     "@angular/platform-browser-dynamic": "~9.1.6",
     "@angular/router": "~9.1.6",
-    "@microsoft/mgt": "^1.3.0",
+    "@azure/msal-angular": "^1.0.0",
+    "@microsoft/mgt": "*",
+    "@microsoft/mgt-element": "*",
```

Comment:
@shweaver-MSFT should we update the [Readme](https://github.com/microsoftgraph/microsoft-graph-toolkit/blob/7dd67f99c6b693d9164691ddab4be95712511893/samples/angular-app/README.md)?

`npm install`
      to
 `yarn install`, npm seems to complain about the registry for `"@microsoft/mgt-element": "*", `

Reply:
Ya, it should be yarn. I'll update

Reply:
Updated!